src/utils/devCourseData.ts

```diff
@@ -0,0 +1,78 @@
+import { devCourseDataType } from "@/types/devCourseData";
+export const devCourseData: devCourseDataType[] = [
+  {
+    id: 482193,
+    title: "Machine Learning A-Z™: Hands-On Python & R In Data...",
+    category: "design",
+    categoryColor: "#993D20",
+    categoryBgColor: "#FFEEE8",
+    price: 57,
+    thumbnail: "/Courses/machine-learning.png",
+    review: 5.0,
+    registeredStudents: 212546,
```

Comment:
Yeni course datasına ihtiyacımız yok mevcut bütün kurslar için bulunan dosyamız data olarak kullanılabilir